### DISCUSSION TITLE
Why the version 13.5 of OpenAM has not been released and there are unbuilt RC1 and RC2 versions?

### DISCUSSION THREAD

**Discussion by korah91** [OpenIdentityPlatform/OpenAM#816] [Q&A] [answered]
Version 13.5 of OpenAM does not appear as released, and if I search in the Tags tab [here ](https://github.com/OpenIdentityPlatform/OpenAM/tags?after=14.0.0-M2), I see that there are two versions of OpenAM 13.5: RC1 and RC2. Is there any reason why it has not been built as a release? 
Regards

**Comment by vharseko** [comment]
[FORGEROCK has shuttered the open-source community, and no longer allows new development on their platform under a permissive license](https://web.archive.org/web/20170902192154/http://www.timeforafork.com/) use [upgrade policy](https://github.com/OpenIdentityPlatform/OpenAM/blob/master/SECURITY.md)

**Reply by korah91** [reply]
Thanks for the response.
Does this mean that I cannot build from source without having permission to the dependencies that are in the Forgerock private repo? I have been trying to access it without my account and it says that I dont have permission. Reading more about it it seems that maybe I need a support license?

I have tried with 13.5 OpenAM and also with 14.1, when I do mvn install in both of them I get similar errors. I'm following [this part ](https://doc.openidentityplatform.org/openam/dev-guide/chap-customizing#build-a-sample-plugin)of the doc

**Comment by vharseko** [comment, marked as answer]
use [HowTo build 14.0.1](https://github.com/OpenIdentityPlatform/OpenAM/tree/14.0.1#how-to-build)
or [HowTo build last version](https://github.com/OpenIdentityPlatform/OpenAM#how-to-build-openam-from-source)
version 13.5 unsupported
